src/main/java/org/folio/consortia/repository/TenantDetailsRepository.java

```diff
@@ -0,0 +1,15 @@
+package org.folio.consortia.repository;
+
+import org.folio.consortia.domain.dto.TenantDetails;
+import org.folio.consortia.domain.entity.TenantDetailsEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.transaction.annotation.Transactional;
+
+public interface TenantDetailsRepository extends JpaRepository<TenantDetailsEntity, String> {
+  @Transactional
```

Comment:
I think lets use transactional annotation on service layer instead of dao layer as we do on other places, or there is specific reason to use on dao layer?

Reply:
Updated. No specific reason, in first implementation there was no separate method for update in service, so used it here. Now there's, so it's better to move it as you say